src/index.js

```diff
@@ -553,6 +554,7 @@ function createConfig(options, entry, format, writeMeta) {
 						keepName: true,
 						keepImport: true,
 					}),
```

Comment:
I'm thinking that maybe it's better to add `svg` to `exclude` property for `rollup-plugin-smart-asset`
It doesn't generate the require `require("./search~ldYgDWJT.svg");` that is breaking in Jest because of `~` sign
This is due to this line https://github.com/facebook/create-react-app/pull/6706/files#diff-5a6116b1d488e7c17de0d52cb4cfd2f8R24